releases/release-5.3.0.md

```diff
@@ -0,0 +1,392 @@
+    - Since v5.3.0, TiCDC and BR support [global temporary tables](/temporary-tables.md#global-temporary-tables). If you use TiCDC and BR of a version earlier than v5.3.0 to replicate global temporary tables to the downstream, a table definition error occurs.
+    - The following clusters are expected to be v5.3.0 or later; otherwise, data error is reported when you create a global temporary table: 
+    
+        - the cluster to be imported using TiDB ecosystem tools
+        - the cluster restored using TiDB ecosystem tools
+        - the downstream cluster in a replication task using TiDB ecosystem tools
+    - For the compatibility information of temporary tables, refer to [Compatibility with MySQL temporary tables](/temporary-tables.md#compatibility-with-mysql-temporary-tables) and [Compatibility restrictions with other TiDB features](/temporary-tables.md#compatibility-restrictions-with-other-tidb-features).
+
+- Fix the issue that the `SHOW VIEW` permission is not required to execute `SHOW CREATE VIEW`. Now you are expected to have the `SHOW VIEW` permission to execute the `SHOW CREATE VIEW` statement.
+- The system variable `sql_auto_is_null` is added to the noop functions. When `tidb_enable_noop_functions = 0/OFF`, modifying this variable value causes an error.
+- The `GRANT ALL ON performance_schema.*` syntax is no longer permitted. If you execute this statement in TiDB, an error occurs.
+- Fix the issue that auto-analyze is unexpectedly triggered outside the specified time period when new indexes are added before v5.3.0. In v5.3.0, after you set the time period through the `tidb_auto_analyze_start_time` and `tidb_auto_analyze_end_time` variables, auto-analyze is triggered only during this time period.
+- The default storage directory for plugins is changed from "" to /data/deploy/plugin.
+- The DM code is migrated to [the folder "dm" in TiCDC code repository](https://github.com/pingcap/ticdc/tree/master/dm). Now DM follows TiDB in version numbers. Next to v2.0.x, the new DM version is v5.3.0, and you can upgrade from v2.0.x to v5.3.0 without any risk.
+
+## New features
+
+### SQL
+
+- **Use SQL interface to set placement rules for data (experimental)**
+
+    Support the `[CREATE | ALTER] PLACEMENT POLICY` syntax that provides a SQL interface to set placement rules for data. Using this feature, you can specify tables and partitions to be scheduled to specific regions, data centers, racks, or hosts. This meets your application demands for lower cost and higher flexibility. The typical user scenarios are as follows:
+
+    - Merge multiple databases of different applications to reduce the cost on database maintenance
```

Comment:
This one is true, but we have to be a little bit careful because it only applies in some circumstances. DDL wil be a limitation because it is only performed by the DDL owner, and schema changes are non-concurrent.

Reply:
This is provided by @ThomasYYYY . What's your suggested way of saying this?  Or can we simply remove this one?

Reply:
@easonn7 Would you please take a look at the above comment from @morgo and suggest the changes? Thanks

Reply:
For an experimental feature, I think there is no problem with the description here. What we should do is to to define these boundary issues clearly in the next formal GA.